part-spec/sensor.json

```diff
@@ -0,0 +1,240 @@
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                }
+            }
+        },
+        {
+            "title": "gyro",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "gyroRanges": {
+                    "description": "range of angular speed that gyro can measure",
+                    "examples": [
+                        "125",
+                        "250",
+                        "500"
+                    ],
+                    "comment": "units of degrees-per-second (dps)",
+                    "type": "number"
+                },
+                "gyroSensitivity": {
+                    "description": "smallest change in angular speed gyro is able to capture (typical). Corresponds to levels in gyro ranges",
+                    "examples": [
+                        "16",
+                        "32",
+                        "65"
+                    ],
+                    "comment": "units of LSB/dps",
+                    "type": "number",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "gyroSensitivityOverTemperature": {
+                    "description": "gyroscope sensitivity change over temperature",
+                    "comment": "units of percentage per degree celsius",
+                    "$ref": "definitions.json#/value"
+                },
+                "axis": {
+                    "description": "number of axes of measurement",
+                    "type": "number"
+                },
+                "zeroRateOffset": {
+                    "description": "output of the gyroscope when no angular velocity is applied",
+                    "comment": "units of degrees-per-second",
+                    "$ref": "definitions.json#/value"
+                },
+                "zeroRateOffsetOverTemperature": {
+                    "description": "gyro zero rate offset change over temperature",
+                    "comment": "units of degrees-per-second per degree celsius (dps/Celcius)",
+                    "$ref": "definitions.json#/value"
+                },
+                "interface": {
+                    "description": "interface(s) for communication to host",
+                    "examples": [
+                        "i2c",
+                        "spi",
+                        "analog"
+                    ],
+                    "type": "string"
```

Comment:
Ditto here about making tihs an array if we expect multiple interfaces

Reply:
changed to array